### PR TITLE
!feat: Add query server

### PR DESCRIPTION
**Pull request by neitdung**

### CODE REVIEW

File: x/multi-staking/keeper/grpc_query_test.go [hunk 1]
```diff
@@ -0,0 +1 @@
+package keeper_test
```
Comment (catShaark) [inline x/multi-staking/keeper/grpc_query_test.go:1]: should not include this if empty

File: proto/multistaking/v1/query.proto [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      returns (QueryValidatorMultiStakingCoinResponse) {
+    option (google.api.http).get =
+        "/realiotech/multistaking/v1/validator-coin/{validator_addr}";
+  }
+  rpc BondWeight(QueryBondWeight) returns (QueryBondWeightResponse) {
+    option (google.api.http).get = "/realiotech/multistaking/v1/weight/{denom}";
+  }
+}
+
+message QueryMultiStakingLocks {
```
Comment (catShaark) [inline proto/multistaking/v1/query.proto:42]: we should do query via ID

File: proto/multistaking/v1/query.proto [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+message QueryMultiStakingLocksResponse {
+  repeated MultiStakingLock locks = 1;
+  cosmos.base.query.v1beta1.PageResponse pagination = 2;
+}
+
+message QueryMultiStakingLock {
+  option (gogoproto.equal) = false;
+  option (gogoproto.goproto_getters) = false;
+  string delegator_addr = 1 [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
```
Comment (catShaark) [inline proto/multistaking/v1/query.proto:54]: this should be lockID

File: proto/multistaking/v1/query.proto [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+message QueryMultiStakingUnlocksResponse {
+  repeated MultiStakingUnlock unlocks = 1;
+  cosmos.base.query.v1beta1.PageResponse pagination = 2;
+}
+
+message QueryMultiStakingUnlock {
+  option (gogoproto.equal) = false;
+  option (gogoproto.goproto_getters) = false;
+  string delegator_addr = 1 [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
```
Comment (catShaark) [inline proto/multistaking/v1/query.proto:75]: this should be unlockID

File: x/multi-staking/keeper/grpc_query_test.go (hunk 1 above)
Comment (neitdung) [reply x/multi-staking/keeper/grpc_query_test.go]: I've remove this

File: proto/multistaking/v1/query.proto (hunk 2 above)
Comment (neitdung) [reply proto/multistaking/v1/query.proto:42]: We should use string for easy developing at FE side, an url shouldn't have struct inside itself.

Comment (catShaark) [reply proto/multistaking/v1/query.proto:42]: then it should be multistaker_addr and val_addr

File: proto/multistaking/v1/query.proto (hunk 4 above)
Comment (catShaark) [reply proto/multistaking/v1/query.proto:75]: should be multistaker and val addr

Comment (neitdung) [reply proto/multistaking/v1/query.proto]: I've changed param names of 2 message please review it @catShaark

File: proto/multistaking/v1/query.proto (hunk 2 above)
Comment (catShaark) [reply proto/multistaking/v1/query.proto:42]: @neitdung

Comment (catShaark) [reply proto/multistaking/v1/query.proto:42]: ohh sorry I misread this, it's query locks not a single lock

File: proto/multistaking/v1/query.proto [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+message QueryMultiStakingLocks {
+  cosmos.base.query.v1beta1.PageRequest pagination = 1;
+}
+
+message QueryMultiStakingLocksResponse {
+  repeated MultiStakingLock locks = 1;
+  cosmos.base.query.v1beta1.PageResponse pagination = 2;
+}
+
+message QueryMultiStakingLock {
```
Comment (catShaark) [inline proto/multistaking/v1/query.proto:51]: should it be `QueryMultiStakingLockRequest`

**Pull request merged**